core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -541,20 +541,23 @@ private[ui] class JobPagedTable(
 
   override def headers: Seq[Node] = {
     // Information for each header: title, cssClass, and sortable
-    val jobHeadersAndCssClasses: Seq[(String, String, Boolean)] =
+    val jobHeadersAndCssClasses: Seq[(String, String, Boolean, String)] =
```

Comment:
Should this be Option[String]? seems like below you'd make an empty tooltip for the rest.